models/topics.models.js

```diff
@@ -0,0 +1,5 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db.query(`SELECT * FROM topics`).then(({ rows: topics }) => topics);
```

Comment:
Be careful, you are missing the ; after topics

Reply:
Cheers for the review, will fix this then merge.